set.mm

```diff
@@ -20200,13 +20201,12 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
     $( Define temporary individual variables. $)
     cbvex4vw.vf $f setvar f $.
     cbvex4vw.vg $f setvar g $.
-    $d w z ch $.  $d u v ph $.  $d x y ps $.  $d f g ps $.  $d f w $.
-    $d v x $.  $d f z $.  $d g z $.  $d u v w z $.  $d u w x z $.
-    $d v w y z $.  $d w x y z $.  $d u y $.  $d g w $.
+    $d w z ch $.  $d u v ph $.  $d x y ps $.  $d f g ps $.  $d f g w z $.
+    $d u v w x y z $.
     cbvex4vw.1 $e |- ( ( x = v /\ y = u ) -> ( ph <-> ps ) ) $.
     cbvex4vw.2 $e |- ( ( z = f /\ w = g ) -> ( ps <-> ch ) ) $.
-    $( Version of ~ cbvex4v with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    $( Version of ~ cbvex4v with more disjoint variable condition, which does
+       not require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
```

Comment:
```suggestion
    $( Version of ~ cbvex4v with more disjoint variable conditions, which
       requires fewer axioms .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
```
In case you accept my previous suggestion, this automatically drops ax-11 and ax-12 as well.